### DISCUSSION TITLE
SEQ stopped logging suddenly

### DISCUSSION THREAD

**Discussion by Verbient** [datalust/seq-tickets#1371] [Help]
Hi,

The only messages SEQ showed for quite some time now is "Starting Up".
Any reason this is happening? Serilog is logging properly in SQL server and logs are generated, but I don't understand why SEQ suddenly stopped logging. Volume of logging is also trivial as the site has not yet been launched fully, hence there should be no question of insufficient memory.

Please check a fraction of the latest SEQ logs as follows:

{"@t":"2021-11-05T14:40:33.5271209Z","@mt":"Native storage metrics sampled","ABI_INTERNAL_ERROR":0,"ABI_IO_ERROR":0,"ABI_PANIC":0,"DELETE_ABANDON":0,"DELETE_BEGIN":0,"DELETE_COMPLETE":0,"DELETE_EXCLUDE_EVENT":0,"DELETE_SKIP_SPAN":0,"INDEX_SET_CREATE":0,"INDEX_SET_LOAD":0,"INDEX_SET_MISSING_INDEX_LOOKUP":0,"INDEX_SET_REMOVE":0,"INDEXER_ABORT":0,"INDEXER_BEGIN_NEW":0,"INDEXER_BEGIN_OUTDATED":0,"INDEXER_COALESCED":0,"INDEXER_COMMIT":0,"INGEST_BUF_CREATE":24,"INGEST_BUF_READ":43,"INGEST_BUF_READ_TRUNCATED":0,"INGEST_BUF_REMOVE":0,"INGEST_BUF_WRITE_EVENT":24,"MAINTENANCE_TRANSACTION_BEGIN":125,"MAINTENANCE_TRANSACTION_COMPLETE":125,"MAINTENANCE_TRANSACTION_DIRTY_COMPLETE":0,"METADATA_READ_TRUNCATED":0,"QUERY_BEGIN":1871,"QUERY_END":1871,"QUERY_ROW_HIT":102905,"QUERY_ROW_MISS_FRAGMENT":0,"QUERY_ROW_MISS_PREDICATE":0,"READ_TRANSACTION_BEGIN":141,"READ_TRANSACTION_COMPLETE":141,"READ_TRANSACTION_DIRTY_COMPLETE":0,"REPLICATION_FULL_SYNC":0,"STABLE_SPAN_CREATE":0,"STABLE_SPAN_LAST_SEEK_PROBES":0,"STABLE_SPAN_READ":0,"STABLE_SPAN_READ_NO_INDEX_SET":0,"STABLE_SPAN_READ_SKIP":0,"STABLE_SPAN_REMOVE":0,"STABLE_SPAN_SEEK":0,"STABLE_SPAN_SPARSE_READ":0,"STORE_CLOSE":0,"STORE_CREATE":0,"STORE_INGEST_BUF_READ":128,"STORE_OPEN":1,"TRUNCATE_COMPLETE":0,"WRITE_TRANSACTION_BEGIN":24,"WRITE_TRANSACTION_COMPLETE":24,"WRITE_TRANSACTION_DIRTY_COMPLETE":0,"WRITER_ABORT":0,"WRITER_BEGIN":24,"WRITER_COMMIT":24,"WRITER_COMMIT_MULTIPLE_BUFS":0}
{"@t":"2021-11-05T14:40:34.0275908Z","@mt":"Applying {Count} retention policies","Count":0,"SourceContext":"Seq.Server.Features.Retention.RetentionPolicyApplier"}
{"@t":"2021-11-05T14:40:34.0277088Z","@mt":"Retention processing and compaction took {Elapsed} ms; allocating {Allocation} ms for indexing","Elapsed":0.2006,"Allocation":599999.7994}
{"@t":"2021-11-05T14:40:34.0277493Z","@mt":"Beginning indexing","SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T14:40:34.0305540Z","@mt":"Wrote {IndexSetCount} index sets in {Elapsed:0.000} ms","@r":["2.000"],"IndexSetCount":0,"Elapsed":2,"SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T14:40:34.0305938Z","@mt":"Indexing took {Elapsed} ms; total iteration time was {Iteration} ms","Elapsed":2,"Iteration":3.045}
{"@t":"2021-11-05T14:40:35.1833701Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.6438811764088722,"ProcessWorkingSetBytes":159272960,"ProcessThreads":32,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}
{"@t":"2021-11-05T14:45:35.1908111Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.6519216554039553,"ProcessWorkingSetBytes":159236096,"ProcessThreads":31,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}
{"@t":"2021-11-05T14:50:34.0293669Z","@mt":"Applying {Count} retention policies","Count":0,"SourceContext":"Seq.Server.Features.Retention.RetentionPolicyApplier"}
{"@t":"2021-11-05T14:50:34.0295584Z","@mt":"Retention processing and compaction took {Elapsed} ms; allocating {Allocation} ms for indexing","Elapsed":0.2798,"Allocation":599999.7202}
{"@t":"2021-11-05T14:50:34.0295952Z","@mt":"Beginning indexing","SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T14:50:34.0337393Z","@mt":"Wrote {IndexSetCount} index sets in {Elapsed:0.000} ms","@r":["4.000"],"IndexSetCount":0,"Elapsed":4,"SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T14:50:34.0338387Z","@mt":"Indexing took {Elapsed} ms; total iteration time was {Iteration} ms","Elapsed":4,"Iteration":4.5176}
{"@t":"2021-11-05T14:50:35.2006210Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.6514399896990304,"ProcessWorkingSetBytes":159617024,"ProcessThreads":32,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}
{"@t":"2021-11-05T14:55:35.1988541Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.6459690303258636,"ProcessWorkingSetBytes":160104448,"ProcessThreads":30,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}
{"@t":"2021-11-05T15:00:34.0340158Z","@mt":"Applying {Count} retention policies","Count":0,"SourceContext":"Seq.Server.Features.Retention.RetentionPolicyApplier"}
{"@t":"2021-11-05T15:00:34.0341251Z","@mt":"Retention processing and compaction took {Elapsed} ms; allocating {Allocation} ms for indexing","Elapsed":0.196,"Allocation":599999.804}
{"@t":"2021-11-05T15:00:34.0341619Z","@mt":"Beginning indexing","SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T15:00:34.0383102Z","@mt":"Wrote {IndexSetCount} index sets in {Elapsed:0.000} ms","@r":["4.000"],"IndexSetCount":0,"Elapsed":4,"SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T15:00:34.0384058Z","@mt":"Indexing took {Elapsed} ms; total iteration time was {Iteration} ms","Elapsed":4,"Iteration":4.4353}
{"@t":"2021-11-05T15:00:35.2052908Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.6471250280176833,"ProcessWorkingSetBytes":160202752,"ProcessThreads":32,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}
{"@t":"2021-11-05T15:05:35.2177222Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.6482237981009972,"ProcessWorkingSetBytes":160116736,"ProcessThreads":34,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}
{"@t":"2021-11-05T15:10:34.0443791Z","@mt":"Applying {Count} retention policies","Count":0,"SourceContext":"Seq.Server.Features.Retention.RetentionPolicyApplier"}
{"@t":"2021-11-05T15:10:34.0445464Z","@mt":"Retention processing and compaction took {Elapsed} ms; allocating {Allocation} ms for indexing","Elapsed":0.3181,"Allocation":599999.6819}
{"@t":"2021-11-05T15:10:34.0445839Z","@mt":"Beginning indexing","SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T15:10:34.0475748Z","@mt":"Wrote {IndexSetCount} index sets in {Elapsed:0.000} ms","@r":["2.000"],"IndexSetCount":0,"Elapsed":2,"SourceContext":"Seq.Server.Features.Indexing.EventStoreIndexer"}
{"@t":"2021-11-05T15:10:34.0476423Z","@mt":"Indexing took {Elapsed} ms; total iteration time was {Iteration} ms","Elapsed":3,"Iteration":3.3715}
{"@t":"2021-11-05T15:10:35.2150553Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.6619679620771715,"ProcessWorkingSetBytes":160059392,"ProcessThreads":32,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}
{"@t":"2021-11-05T15:15:35.2260534Z","@mt":"Metrics sampled","CacheDays":1.5,"CacheEvents":191,"SystemMemoryUtilization":0.7453810166484651,"ProcessWorkingSetBytes":160083968,"ProcessThreads":35,"ArrivedEventsPerMinute":0,"IngestedEventsPerMinute":0,"IngestedBytesPerMinute":0,"InvalidPayloadsPerMinute":0,"UnauthorizedPayloadsPerMinute":0,"BroadcastQueueCount":0,"BroadcastSubscriptionQueuesAggregateCount":0}

**Comment by nblumhardt** [comment]
Thanks for the note! Have you checked the ingestion log? It's accessible via Settings > Diagnostics > Ingestion Log.

There are a few more notes on ingestion troubleshooting in: https://docs.datalust.co/docs/ingestion-troubleshooting - my best guess is that it will be an event payload size issue, which would normally leave a message in the ingestion log.

Let me know if this helps! 🤞

**Reply by Verbient** [reply]
I checked the documentation and the image under "3. Check the Ingestion Screen".  There is the following statement: "If payloads are being rejected for any reason, a link to the Ingestion Log will appear beside the counter with details of the problem." What does this mean? You should have depicted this in the image. Consider making the documentation little more verbose. You could  also add few video tutorials, which would be really nice.
I also mailed seq support team with a copy of seq diagnostic log. Hope they come up with a cause.

**Reply by Verbient** [reply]
Apologies, but I must say that you should improve on your support process. Liam responded to me on mail after more than 2 days and yet there was no resolution.

**Reply by nblumhardt** [reply]
Hi Sujoy, sorry to hear you're still having trouble!

I see that Liam asked for your Serilog `SelfLog` output, but I don't have a copy of your reply; I'll sync with Liam when he's online in case I wasn't copied, but if you haven't sent that through yet, could you please forward it along? `SelfLog` will show up a lot of networking-type issues that Seq won't be able to show in the ingestion log.

**Reply by Verbient** [reply]
Apologies, it was an error on my part. The server URl was not correct in appsetting.json it was overwritten during one of the publish. I fixed the issue. Also I have now set up a Serilog selflog. Thanks for your help :) .
